components/servo/webview_delegate.rs

```diff
@@ -454,6 +509,14 @@ pub trait WebViewDelegate {
     /// Request to stop a haptic effect on a connected gamepad.
     fn stop_gamepad_haptic_effect(&self, _webview: WebView, _: usize, _: IpcSender<bool>) {}
 
+    fn show_select_element_prompt(
+        &self,
+        _webview: WebView,
+        select_element_prompt: SelectElementPrompt,
+    ) {
+        select_element_prompt.submit();
+    }
+
```

Comment:
I suspect we will be needing to show other kinds of forms controls, such as date and color pickers. Perhaps this could be a more generic API that uses an enum like the one we use for dialogs. I'm imagining something like `show_form_control`.

Reply:
Hmm these don't really share anything in common though, besides the screen area of the form control.

Reply:
The idea would be that there would be a single delegate method to handle showing all form controls (and perhaps a fallback that uses HTML and CSS). The benefit here is that someone embedding Servo can look at the enum passed to the `show_form_control` delegate method and know what needs to be implemented in order to support the web. This is how the `show_simple_dialog` method works as well.

Reply:
Yeah that makes sense. Thanks for the explanation (:

Reply:
Done